src/common/io/tls/client.c

```diff
@@ -340,7 +355,8 @@ tlsClientNew(SocketClient *socket, TimeMSec timeout, bool verifyPeer, const Stri
         *driver = (TlsClient)
         {
             .memContext = MEM_CONTEXT_NEW(),
-            .socketClient = sckClientMove(socket, MEM_CONTEXT_NEW()),
+            .ioClient = ioClientMove(ioClient, MEM_CONTEXT_NEW()),
+            .host = strDup(host),
```

Comment:
This seems a bit grotty, do we actually need two copies of it...?  What if one is changed?  Which is canonical..?

Reply:
Well, this is the host we are going to use to verify TLS -- there's no reason that has to be the same host we pass to SocketClient, though that's the case now.

Reply:
We should figure out a way to make this cleaner because we need to make absolutely sure that the host we're using for verification is the exact value of what was configured in the pgbackrest.conf.